packages/ds-core/src/colors.scss

```diff
@@ -3,8 +3,8 @@
 
 // Primary UI Colors:
 
-$dark-blue:                 #0086c3 !default;
-$basic-blue:                #00a1e5 !default;
+$dark-blue:                 #0188E1 !default;
```

Comment:
1. why were all those package-lock.js files added? is there a need to commit them?
2. don't forget to also bump the version